README.md

```diff
@@ -1,8 +1,9 @@
-# PFRPG: Clock Adjuster
+# CoreRPG, 3.5E, 5E, PFRPG, PF2RPG: Clock Adjuster v2.5
 This extension is a GM tool which provides an interface for easy time tracking and management.
 
 # Compatibility and Instructions
-This extension has been tested with [FantasyGrounds Unity](https://www.fantasygrounds.com/home/FantasyGroundsUnity.php) 4.1.4 (2021-06-08).
+This extension has been tested with [FantasyGrounds Unity](https://www.fantasygrounds.com/home/FantasyGroundsUnity.php) 4.1.4 (2021-06-08) and
+[FantasyGrounds Classic](https://www.fantasygrounds.com/home/FantasyGroundsClassic.php) 3.3.15.
```

Comment:
I'm a Classic user and tested on both.  Also validated that the nextRound override is equivalent for both versions.